pkg/api/templates.go

```diff
@@ -45,7 +45,7 @@ type TemplateResponse struct {
 
 // We use a separate struct because version and arch cannot be updated
 type TemplateUpdateRequest struct {
-	UUID            *string        `json:"uuid" readonly:"true" swaggerignore:"true"`
+	UUID            *string        `json:"uuid" readonly:"true" swaggerignore:"true" validate:"required"`
```

Comment:
When updating the template, the uuid comes from the url param and not from the request body, so I wouldn't mark it as required here